app/src/main/java/com/github/se/cyrcle/ui/parkingDetails/ParkingDetailsAlertDialogShowImage.kt.kt

```diff
@@ -0,0 +1,71 @@
+package com.github.se.cyrcle.ui.parkingDetails
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
+import androidx.compose.material3.BasicAlertDialog
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import coil.compose.rememberAsyncImagePainter
+import com.github.se.cyrcle.ui.theme.atoms.IconButton
+
+/**
+ * Alert dialog to show the image of a parking spot. This composable is displayed when the user
+ * clicks on an image of a parking spot.
+ *
+ * @param onDismiss Callback when the dialog is dismissed.
+ * @param imageUrl URL of the image to display.
+ */
+@Composable
+@OptIn(ExperimentalMaterial3Api::class)
+fun ParkingDetailsAlertDialogShowImage(onDismiss: () -> Unit, imageUrl: String) {
+  val alertDialogMaxHeight = LocalConfiguration.current.screenHeightDp.dp * 0.8f
```

Comment:
minor detail but you can create a val for LocalConfiguration.current.screenHeightDp.dp instead of calling it twice.

Reply:
Yes,  I think I will just remove the second time I called it because it is useless.